content/test/integration/denylist/NoopDenylist.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { Bean, EnvironmentConfig } from '@katalyst/content/Environment'
+import { assertPromiseIsRejected } from '@katalyst/test-helpers/PromiseAssertions'
+import { MockedContentCluster } from '@katalyst/test-helpers/service/synchronization/MockedContentCluster'
+import { MockedSynchronizationManager } from '@katalyst/test-helpers/service/synchronization/MockedSynchronizationManager'
+import { loadStandaloneTestEnvironment } from '../E2ETestEnvironment'
+import { buildDeployData, createIdentity } from '../E2ETestUtils'
+import { TestServer } from '../TestServer'
+
+describe('Integration - NoopDenylist', () => {
+  const metadata: string = 'Some metadata'
+  const decentralandIdentity = createIdentity()
+  const ownerIdentity = createIdentity()
+  const testEnv = loadStandaloneTestEnvironment()
+  let server: TestServer
+
+  beforeEach(async () => {
+    server = await testEnv
+      .configServer()
+      .withBean(Bean.SYNCHRONIZATION_MANAGER, new MockedSynchronizationManager())
```

Comment:
I think you can just use the config that disables the sync instead of adding the mock